gateway/assets/error.html

```diff
@@ -0,0 +1,86 @@
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18.4 21"><circle cx="7.5" cy="4.8" r="1"/><circle cx="11.1" cy="4.8" r="1"/><path d="M12.7 8.4c-0.5-1.5-1.9-2.5-3.5-2.5 -1.6 0-3 1-3.5 2.5H12.7z"/><path d="M8.5 9.7H5c-0.5 0.8-0.7 1.7-0.7 2.7 0 2.6 1.8 4.8 4.2 5.2V9.7z"/><path d="M13.4 9.7H9.9v7.9c2.4-0.4 4.2-2.5 4.2-5.2C14.1 11.4 13.9 10.5 13.4 9.7z"/><circle cx="15.7" cy="12.9" r="1"/><circle cx="15.1" cy="15.4" r="1"/><circle cx="15.3" cy="10.4" r="1"/><circle cx="2.7" cy="12.9" r="1"/><circle cx="3.3" cy="15.4" r="1"/><circle cx="3.1" cy="10.4" r="1"/></svg>
+      </a>
+    </div>
+  </div>
+</div>
+
+  <div id="content">
+    <div id="content-header">
+      <strong>{{ .StatusCode }} {{ .StatusText }}</strong>
+    </div>
+
+    <div class="table-responsive">
+      <table>
+        <tr>
+          <td>
+            {{ if eq .StatusCode 400 }}
+              <p>Your request is invalid. Please check the error below for more information.</p>
+            {{ else if eq .StatusCode 404 }}
+              <p>The content path you requested cannot be found. There's likely an invalid or missing DAG node.</p>
+            {{ else if eq .StatusCode 410 }}
+              <p>The content path you tride to request is correct, but this gateway refuses to return the requested data.</p>
+            {{ else if eq .StatusCode 412 }}
+              <p>This gateway is unable to return the requested data under the conditions sent by the client.</p>
+            {{ else if eq .StatusCode 429 }}
+              <p>You have made too many requests. Please wait some time and try again.</p>
+            {{ else if eq .StatusCode 451 }}
+              <p>This gateway is not allowed to return the requested data due to legal reasons.</p>
+            {{ else if eq .StatusCode 500 }}
+              <p>This gateway was unable to return the requested data due to an internal error. Please check the error below for more information.</p>
+            {{ else if eq .StatusCode 504 }}
+              <p>This gateway was unable to fullfil your request within internally defined time limits.</p>
+            {{ end }}
+
+            <p><code>{{ .Error }}</code></p>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <p>How you can proceed:
+              <ul>
+                <li>Check the <a href="https://discuss.ipfs.tech/c/help/13" rel="nofollow">Discussion Forums</a> for similar errors</li>
+                <li>Try diagnosing your request with the <a href="https://docs.ipfs.tech/reference/diagnostic-tools/" rel="nofollow">diagnostic tools</a></li>
+                <li>Self-host and run a client that verifies your data, such as <a href="https://docs.ipfs.io/install/command-line/" rel="nofollow">Kubo</a>, <a href="https://github.com/ipfs/ipfs-desktop" rel="nofollow">IPFS Desktop</a>, or <a href="https://brave.com/ipfs-support/" rel="nofollow">Brave Browser</a></li>
```

Comment:
Rather than being opinionated about an installation to use, maybe just link to https://docs.ipfs.tech/concepts/ipfs-implementations/ ?

Reply:
I feel that docs page is not the best from the perspective of a regular user reading this in a web browser.

Things in "Popular" are all backend daemons, normal people will just bounce off that website, and we lose potential users who could do just fine with IPFS Desktop or Brave.

Maybe add  "Looking for User Friendly Implementation?" section at the top with IPFS Desktop+IPFS Companion and  [Brave](https://brave.com/ipfs-support/)  options?

Reply:
@lidel  this suggestion is to update the Docs page or to update the error template?

Reply:
@lidel I assume that your idea is to update the "IPFS Implementations" page in [ipfs-docs](https://github.com/ipfs/ipfs-docs) to have a "User Friendly Implementations" section at the top. If so, I do agree with it. I updated the link on this PR to the docs. I can open a PR on the docs if I understood it correctly.

Reply:
Opened issue: https://github.com/ipfs/ipfs-docs/issues/1606